src/components/TopBar.js

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from 'react';
+import './TopBar.css';
+import logo from '../logoTopBar.png';
+
+class TopBar extends Component {
+  constructor() {
+    super();
+    // TODO: Replace these information by the one returned from the OpenID api
```

Comment:
Would you mind removing this comment, please? 🙃

Reply:
Sure, no problem! I thought it would be a good idea to let it as a tip for the next developer 😛 
Already removed that in the last commit